assets/template-collection.css

```diff
@@ -293,7 +293,7 @@
 }
 
 .facets__price .field__label {
-  left: 2.1rem;
+  left: 3rem;
```

Comment:
Spacing still looks off if using a different currency.

USD:
<img src="https://user-images.githubusercontent.com/37168033/128040208-1469a3f5-fec2-4e22-86f9-07bc272c2a1e.png" width="300">

BRL:
<img src="https://user-images.githubusercontent.com/37168033/128040157-7566a092-d8e8-4859-a5fb-f31219d7154b.png" width="300">

Also, not related to this PR but there is a bug with price formatting on this component:
<img src="https://user-images.githubusercontent.com/37168033/128040386-3f93a786-22a3-4bc1-865a-5d2ae890b226.png" width="300">

We should either `strip_html` or add `_html` to the translation tag.

Reply:
Maybe the currency should be outside of the relatively positioned container so that it can't impact the position of the label 🤔

Reply:
Thank you Lucas for finding the bug!

I followed Tyler's point and put the label out of the wrapper so the position should now be fixed. I also added the `strip_html` to the message.